enterprise-java.html.slim

```diff
@@ -0,0 +1,67 @@
+---
+layout: microsite
+hero_class: enterprise-java
+---
+
+= define 'hero'
+  .row
+    .large-13.column
+      h2 The Place to Come for Enterprise Java
+    .large-8.column
+      p JBoss® Enterprise Application Platform 6 provides an innovative modular, cloud-ready architecture, powerful management and automation, and world class developer productivity. It is Java™ EE 6 certified and features powerful yet flexible management, improved performance and scalability, and many new features to maximize developer productivity. It builds on the community foundation of WildFly.
+
+.wide.microsite-text-points
+  .row
+    .large-6.column
+      h4 Learn About Community
+      asciidoc:  
+		 Work alongside smart, motivated people to apply your skills toward a bigger effort. Look at the link:http://javaee-samples.github.io/[Java EE 7 Samples Project] or contribute to link:https://github.com/jboss-developer/www.jboss.org[JBoss Developer] and don't forget to find solutions in our link:http://www.jboss.org/forums/[Forums].
+    .large-12.column
+      h4 Getting Started Is Easy
+      asciidoc:   
+		Find all you need in the link:http://www.jboss.org/products/eap/get-started/[Getting Started Section] on JBoss developer. It's not much more than just hitting download and unzip into a folder of your choice. And we also provide link:http://www.jboss.org/products/eap/developer-materials[Developer Materials] and link:http://www.redhat.com/en/services/training[Training] to improve your skills, or the skills of your team, with our hands-on, role-based training for IT professionals and developers. If you feel the need for latest innovation and specifications, you need to look at link:http://wildfly.org/[WildFly]. This is the community version and helps you build flexible and lightweight applications.
+    .large-6.column
+	h4 Learn About Technology	
+	asciidoc:     
+		There are many resources available for link:http//:www.jboss.org/products/eap/overview/[Red Hat JBoss Enterprise Application Platform] on link:http://www.jboss.org[JBoss Developer] or the link:http://access.redhat.com[Red Hat Customer Portal]. You can also follow our Developer Advocates link:http://www.arungupta.me[Arun Gupta] or link://blog.eisele.net[Markus Eisele] and see, what they have to share.
```

Comment:
You can abbreviate "Red Hat JBoss Enterprise Application Platform" to "JBoss EAP" if you want.